src/graphics.cpp

```diff
@@ -105,8 +105,6 @@ void GetvScreen(const int A)
             if(vScreen[A].TempY < 0)
                 vScreen[A].TempY += 1;
         }
-        if(p.Mount == 2)
-            pLoc.Height = 128;
```

Comment:
Why you removed this assignment?

Reply:
The purpose of that commit was to avoid using the player location as a temporary variable, but now I see that it's possible that the player's previous location was not 128. So, I'll revert this and leave a note for it.

Reply:
I decided to simply revert the commit. There isn't enough benefit to justify the complication of changing and explaining the case in the code.